build_lookup.sh

```diff
@@ -19,5 +19,9 @@ rm -rf ./txblocks.db
 rm -rf ./test.db
 rm -rf ./txbodies.db
 
+find . -name *.cmake | xargs rm -rf
```

Comment:
💡 Another option is to not have this in the script, but instead run the following to clean *everything* in the working directory:

```
git clean -dxf
```

💡 It's good to make changes like this in their own commits, since it's not directly related to the rest of this pull request.